### ISSUE TITLE
Import all media from device

### ISSUE THREAD

**Issue by alfechner** [whoozle/android-file-transfer-linux#213]
I use Android File Transfer on Linux. I miss a way to automatically import all media files from my mobile to my computer without finding them myself.

I'd like the application browse the mobiles file system and copy any media file to the computer. It should remember what has already been imported. Next time the application will only import the media files not imported yet.

Is this a valid/potential use case for Android File Transfer?

**Comment by whoozle** [comment]
So you think of rsync-like command to keep your phone/computer up to date? Sounds like good idea, actually you can use mtp-mount + rsync combination. aft-mtp-mount is really fast

**Comment by alfechner** [comment]
Bringing `rsync` into play is very interesting. Yes something like that. There is no aspiration to keep Android File Transfer platform independent, it's meant to run on exclusively on Linux, right?

Besides the technical way of implementing something like that I was wondering if you think, that a functionality like that might be a nice feature for Android File Transfer at all.

**Comment by yurikoles** [comment]
iTunes for Android/Linux? :)

**Comment by alfechner** [comment]
@yurikoles yes, kind of. I don't want to manually collect all images and videos from all the different apps all the time. I look for a reliable and easy to use tool which does that for me.
